streaming/base/format/json/reader.py

```diff
@@ -54,6 +54,16 @@ def __init__(
 
     @classmethod
     def from_json(cls, dirname: str, split: Optional[str], obj: Dict[str, Any]) -> Self:
+        """Initialize from JSON object.
+
+        Args:
+            dirname (str): Local directory containing shards.
+            split (Optional[str]): Which dataset split to use, if any.
```

Comment:
Is there any standard or norm do have to follow between `(Optional[str])` and `(str, optional)` ? Do you know of any standardization?

Reply:
Fixed, everywhere